ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/CommitPackageValidationMode.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils
+
+// define the different package validation modes.
+sealed abstract class CommitPackageValidationMode extends Product with Serializable
+
+object CommitPackageValidationMode {
+  // Specify that the package committer should validate the packages
+  // before committing them to the ledger.
+  case object Precommit extends CommitPackageValidationMode
```

Comment:
Can I suggest `PreCommit` and `PostCommit`, with a capitalized "commit"?

Reply:
+1